msteams-platform/samples/app-templates.md

```diff
@@ -687,4 +687,9 @@ Workplace Awards is a Teams app template that provides a positive framework to f
 :::column-end:::
 :::row-end:::
 
-Have an idea for an app template you'd like to see? [Please let us know](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR2_7qFm_lcZAr4eqEhnLsZ9UMVZGT1lCT0FXUDdZMUM0RkpBS1BESTAwWC4u).
+For more information on app template, see [App template](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR2_7qFm_lcZAr4eqEhnLsZ9UMVZGT1lCT0FXUDdZMUM0RkpBS1BESTAwWC4u).
+
```

Comment:
Have a See also section in all pages and list all the cross-reference for more information in the section.